### PR TITLE
Profile dropdown menu will close after avatar is clicked

### LINKED ISSUE #3701: Profile dropown not closing after clicking on it
### Issue Summary
After tapping profile icon when dropdown in open should close it.

### Steps to Reproduce

https://user-images.githubusercontent.com/53316345/182912184-5c8c0e94-332b-4fc4-a5ea-b69cf6a10989.mp4

### PR DESCRIPTION
**Pull request by om-ray**
## What does this PR do?
Clicking on the avatar icon on mobile will now close the profile dropdown if it is already open.

Fixes #3701

**Environment**: Staging(main branch) / Production

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by om-ray** [comment]
@zomars @hariombalhara

**Review by alannnc — APPROVED** [review]
One line change and it looks good to me. Lets test on the deploy branch before merging.

File: packages/ui/v2/core/Shell.tsx [hunk 1]
```diff
@@ -238,7 +238,7 @@ function UserDropdown({ small }: { small?: boolean }) {
   }
   return (
     <Dropdown open={menuOpen}>
-      <DropdownMenuTrigger asChild onClick={() => setMenuOpen(true)}>
+      <DropdownMenuTrigger asChild onClick={() => setMenuOpen(!menuOpen)}>
```
Comment (hariombalhara) [inline packages/ui/v2/core/Shell.tsx:241]: I think `setMenuOpen((menuOpen)=>!menuOpen)` is a logically better See https://stackoverflow.com/a/48394541/1109657

Comment (om-ray) [reply packages/ui/v2/core/Shell.tsx:241]: @hariombalhara You're right, state changes like this should always be a function, thank you for reminding me! I'll make the change and push!

**Pull request merged**